eea/progressbar/listing/catalog.py

```diff
@@ -16,6 +16,7 @@ def progress(self):
         """
         context = self.getObject()
         request = context.REQUEST
+        import pdb; pdb.set_trace()
```

Comment:
I don't think this is supposed to be here :smile:

Reply:
Removed